docs/delegators/how-to-guides/bridge-lpt.md

```diff
@@ -10,31 +10,42 @@ Please note that you will need some ETH in your wallet to complete this guide. I
 
- ![BridgeApprove](https://user-images.githubusercontent.com/89408276/155850572-2337514c-f6ad-419a-a9bf-94e7d3e1d891.jpg)
- 
-10. Click 'Deposit' and confirm the transaction in your wallet to complete the bridging of LPT from L1 to L2
+![BridgeApprove](https://user-images.githubusercontent.com/89408276/155850572-2337514c-f6ad-419a-a9bf-94e7d3e1d891.jpg)
 
- ![Bridge](https://user-images.githubusercontent.com/89408276/155375033-6fd66e8a-53ab-43e9-9fe6-3a0cec847a55.jpg)
+10. Click 'Deposit' and confirm the transaction in your wallet to complete the bridging of LPT from L1 to L2
 
+![Bridge](https://user-images.githubusercontent.com/89408276/155375033-6fd66e8a-53ab-43e9-9fe6-3a0cec847a55.jpg)
 
 ### Bridging from L2 to L1
+
 The same general instructions apply with three differences:
+
 - You should start with your wallet connected to Arbitrum (Arbitrum Rinkeby if using testnet)
 - You will need to manually approve the amount that you are bridging using the [Arbiscan UI](https://arbiscan.io/address/0x289ba1701C2F088cf0faf8B3705246331cB8A839#writeContract). To do so, connect your wallet, scroll to the Approve function, enter the L2LPTGateway address [0x6D2457a4ad276000A615295f7A80F79E48CcD318](https://arbiscan.io/address/0x6D2457a4ad276000A615295f7A80F79E48CcD318) and the amount you'll be transferring and click "Write". Note that the amount is in "Wei", so use a [Unit Converter](https://etherscan.io/unitconverter) (e.g. if you want to bridge 10 LPT, you need to enter 10000000000000000000).
-    - We are looking into options to remove this step. If it seems complex, we recommend swapping to ETH via [Uniswap](https://app.uniswap.org/#/swap?chain=arbitrum) and then transferring the ETH from L2 to L1 instead.
+  - We are looking into options to remove this step. If it seems complex, we recommend swapping to ETH via [Uniswap](https://app.uniswap.org/#/swap?chain=arbitrum) and then transferring the ETH from L2 to L1 instead.
 - Next, click on the arrow below the amount input. The button will switch from "Deposit" to "Withdraw"
 - Click on "Token" -> "Manage token lists" -> enable "Arbed CMC List" and click on "Back to Select Token". You should now be able to choose LPT in the list.
 - After you've sent the withdraw tx, you need to wait ~1 week until you can claim your LPT on L1. You'll see the countdown below. Once the confirmation period is over, you can connect to the Ethereum mainnet and click "Claim" to withdraw your LPT on L1.
 
+### Bridging Failures
+
+The Arbitrum rollup can fail to bridge tokens correctly if the gas prices are fluctuating by too much. This can be caused by gas prices changing from when the original "bridge" transaction was submitted, to when it was sequenced and submitted on L2. In this case, tokens will not reach your L2 wallet after an hour. Arbitrum [provides a page](https://retryable-tx-panel.arbitrum.io/) to check on the status of your L1 deposit transaction:
```

Comment:
Tickets can also fail if the tx runs out of gas. The screenshot attached shows the error for tx running out of gas.

for price fluctuation, it will be better to use this image instead 
![Screenshot 2022-06-30 at 11 22 57 PM](https://user-images.githubusercontent.com/23727056/176745651-98ff56d0-9c0a-4c2d-b9fe-bf3ba1d537a7.png)